cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/cruisecontrol/CruiseControlUserTaskStatus.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.assembly.cruisecontrol;
+
+/**
+ *  Holds the string literals for the possible options of the "Status" field in the JSON returned by the Cruise Control
+ *  REST API /kafkacruisecontrol/user_tasks endpoint.
+ *
+ *  These values are taken directly from the Cruise Control source code for the TaskState enum at:
+ *  com.linkedin.kafka.cruisecontrol.servlet.UserTaskManager.TaskState
+ *  https://github.com/linkedin/cruise-control/blob/master/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java
+ */
+public enum CruiseControlUserTaskStatus {
+
+    ACTIVE("Active"),
+    IN_EXECUTION("InExecution"),
+    COMPLETED("Completed"),
+    COMPLETED_WITH_ERROR("CompletedWithError");
+
+    private String status;
+
+    CruiseControlUserTaskStatus(String status) {
+        this.status = status;
+    }
+
+    @Override
+    public String toString() {
+        return status;
+    }
+
+}
```

Comment:
It seems this class hasn't been referenced yet, this is for future changes, right?

Reply:
While my developing I was following this one (https://github.com/linkedin/cruise-control/blob/master/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskTracker.java) but maybe yours is right?

Reply:
Hummm that is confusing? The values I added to that enum are the ones used buy the method (`getJsonStructure`) which creates the JSON returned by the user task status endpoint. So those (as far as I can see) are the only possible values you can see. The Execution task tracker state might be used internally or by another endpoint?

Reply:
@kyguy Yes this enum is here to make writing the integration tests easier, probably shouldn't have been in this commit but I thought it could help @ppatierno write the operator unit tests.

Reply:
@tomncooper maybe you are right, only real tests will reveal the truth :-)